### ISSUE TITLE
[DataCap Application] <AWS> - <International Neuroimaging Data-Sharing Initiative>

### ISSUE THREAD

**Issue by penko101** [1475Notary/1475-Allocator#14]
### Data Owner Name

Child Mind Institute

### Data Owner Country/Region

Afghanistan

### Data Owner Industry

Life Science / Healthcare

### Website

https://childmind.org/science/

### Social Media Handle

https://x.com/ChildMindInst

### Social Media Type

Slack

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

5PiB

### Expected size of single dataset (one copy)

410TiB

### Number of replicas to store

10

### Weekly allocation of DataCap requested

512TiB

### On-chain address for first allocation

f1l2aidxe4ogjrvc5rhs7dnltwnpuqle6rbcai7by

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
We’re dedicated to transforming the lives of children and families struggling with mental health and learning disorders by giving them the help they need. We’ve become the leading independent nonprofit in children’s mental health by providing gold-standard evidence-based care, delivering educational resources to millions of families each year, training educators in underserved communities, and developing tomorrow’s breakthrough treatments.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
This bucket contains multiple neuroimaging datasets that are part of the International Neuroimaging Data-Sharing Initiative. Raw human and non-human primate neuroimaging data include 1) Structural MRI; 2) Functional MRI; 3) Diffusion Tensor Imaging; 4) Electroencephalogram (EEG) In addition to the raw data, preprocessed data is also included for some datasets.
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

China

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
We would use fil-tool Lotus to convert files into car files. Also we would contact sps to take these files offline. We have enough harddisks to do this.
```

### Please share a sample of the data

```text
aws s3 ls --no-sign-request s3://fcp-indi/
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Sporadic

### For how long do you plan to keep this dataset stored on Filecoin

1.5 to 2 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, South America, Europe, Australia (continent)

### How will you be distributing your data to storage providers

HTTP or FTP server, Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Slack, Big Data Exchange, Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
1. f01955030 Zhejiang
2. f02128256 Canada
3. f01844118 USA
4. f02211576 Guangdong
5. f02114994 Sichuan
```

### How do you plan to make deals to your storage providers

Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by 1475Notary** [comment]
Hello @penko101 ,

I want to ask you some questions.

Is this your first time to apply? Have you ever applied for datacap from another allocator?
Do SPs you work with support retrieval?
Do you have enough token for your plan?
For bookkeeping, I'll keep a record of our communications. Do you accept that?

Also, please send your email to slack: 1475Notary, we need check your identity.

**Comment by penko101** [comment]
@1475Notary 
Yes, but we have experience about Fil+ and we've joined our partner's item.
Yes, we will command sps about retrieval.
I think sps have enough.
Sure!
![image](https://github.com/user-attachments/assets/f065c03b-51d4-4d6b-a4d8-8196120456ac)
I sent the message to your slack.
If you need more, please leave message to me!

**Comment by 1475Notary** [comment]
@penko101 ok I saw it. If you need to change sps in the process, please put the lateset sp list in the application.

**Comment by penko101** [comment]
@1475Notary Thank you notary. I will keep updating on my application. Please approve my application.

**Comment by 1475Notary** [comment]
@penko101 Give you a chance and support your this round.

**Comment by penko101** [comment]
@1475Notary Thank you notary.

**Comment by penko101** [comment]
@1475Notary Dear notary, we add 2 sps for our storage.
f03157879 Los Angeles
f03159626 Dulles

**Comment by 1475Notary** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
There are individual nodes that cannot be connected, please check. 
Other results are still OK.

**Comment by penko101** [comment]
@1475Notary Dear notary, we come to update some details about sps. 
>@1475Notary Dear notary, we add 2 sps for our storage.
f03157879 Los Angeles
f03159626 Dulles

New sps joined us. All sps have public ip address, and you can check it online.
f03100008 Guangdong
f03100009 HongKong
f03030649 Hebei
f01660795 Guangdong
f01084413 Zhejiang

We know the retrieval rate has decreased, so we changed these sps who do not support retrieval any more. I think it will be back to great result of report next time.

**Comment by 1475Notary** [comment]
@penko101 If these nodes can not recover retrieval, you should change them.

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
@penko101 Hello, just come to ask for your storage situation these days. Are you continuing your storage plan?

**Comment by penko101** [comment]
Yes dear notary, as your advice above, we will only choose sps with good retrieval. @1475Notary

**Comment by 1475Notary** [comment]
@penko101 Good. In view of your previous progress, I'd like to continue to support your storage plan. I will wait for a new report.

**Comment by penko101** [comment]
@1475Notary Dear allocator, we have added sps and replaced the old sps to improve retrieval rate.
f02825282 HongKong
f03224828 UK
f03189917 US
f03216485 Sichuan
f02816097 HongKong
f02826007 Singapore
f03231666 Sichuan
f02827996 Guangdong
f03156722 US
f03100004 Guangdong
f03161261 US
f02826588 Guangdong

**Comment by 1475Notary** [comment]
@penko101 
The amount of sps seems more than you said in you application.
>Number of replicas to store
10

Can you give us more explanation about it?

**Comment by penko101** [comment]
@1475Notary Dear allocator, sorry for our dealy.
This is because we have replaced old sps whose condition of retrieval is not good enough. When we find new sps, we would send a little data to test if they support retrieval. But our final goal is focusing on storing ten replicas.

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
@penko101 Can you explain about sps f03100008, f03100009? Their successful rate is 0 and you have said 
>We know the retrieval rate has decreased, so we changed these sps who do not support retrieval any more. I think it will be back to great result of report next time.

But you still sent data to them several days ago.

**Comment by penko101** [comment]
@1475Notary Dear allocator, it is because that sps tried to repair their retrieval program several days ago. As explained by sps, due to the rules of spark, sp's latest retrieval rate need to have recently packaged the data in order to be displayed on spark. Then we resent some data to these sps. After finding that the retrieval failed again, we stopped sending data to these sps.

**Comment by penko101** [comment]
We add one new partner f03241837-Shanghai. 
![image](https://github.com/user-attachments/assets/a2296bc3-553f-46b4-ad41-fde04ab185b1)

> @1475Notary Dear allocator, it is because that sps tried to repair their retrieval program several days ago. As explained by sps, due to the rules of spark, sp's latest retrieval rate need to have recently packaged the data in order to be displayed on spark. Then we resent some data to these sps. After finding that the retrieval failed again, we stopped sending data to these sps.

**Comment by 1475Notary** [comment]
@penko101 OK. Please keep updating sp in time.

**Comment by 1475Notary** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
@penko101 
>The dataset was declared as 480 TiB in the application, but the report's "Deal Data Replication" section shows 731 TiB in the "Unique Data" column. This discrepancy requires explanation.

Can you explain about it?
